python/py-logilab-hmm/Portfile

```diff
@@ -21,17 +23,19 @@ livecheck.url       http://ftp.logilab.org/pub/hmm/
 livecheck.type      regex
 livecheck.regex     {logilab-hmm-(\d+(?:\.\d+)*)\.tar\.gz}
 
-checksums           md5     81f63768df022714e66f6d471245800a \
-                    sha1    6be2012ab3b7ed6ceac42b48a821f6e194c71f6f \
-                    rmd160  b62f3ac49e4d2681138e55490430c4bdfdf418f6
+checksums           rmd160  b62f3ac49e4d2681138e55490430c4bdfdf418f6 \
```

Comment:
Please provide separate commits for updates and obsolesce.

Reply:
Sorry, I shouldn't read codes when sleepy :/